examples/basic-tracer-node/index.js

```diff
@@ -59,4 +59,7 @@ function doWork(parent) {
 
   // Annotate our span to capture metadata about our operation
   span.addEvent('invoking doWork').end();
+  
+  // End the child span.
+  span.end();
```

Comment:
We are ending the span at line 61 (inline with add event), looks like it is little confusing. I would suggest to remove from there and keep the separate operation to end the span.

Reply:
I missed this in my code review, thanks for pointing it out @mayurkale22